test/test_new_command.rb

```diff
@@ -89,9 +100,11 @@ def site_template
     end
 
     should "force created folder" do
-      capture_stdout { Jekyll::Commands::New.process(@args) }
-      output = capture_stdout { Jekyll::Commands::New.process(@args, "--force") }
-      assert_match(%r!New jekyll site installed in!, output)
+      capture_stdout { Jekyll::Commands::New.process(@args, "--force") }
+      output = Jekyll.logger.messages.last
+      success_message = "New jekyll site #{@title.cyan} " \
+      "installed in #{@full_path.cyan}."
+      assert_includes output, success_message
```

Comment:
The output would be displayed only is the script executed successfully upto that point. So, in theory, it shouldn't matter. `assert_includes` is more tolerant of the ANSI escape codes present in the output.
The context for default path also [checks](https://github.com/jekyll/jekyll/blob/master/test/test_new_command.rb#L45) via an `assert_includes`.